apps/server/src/modules/learnroom/uc/course.uc.ts

```diff
@@ -1,13 +1,27 @@
 import { Injectable } from '@nestjs/common';
-import { EntityId, Course, Counted, SortOrder } from '@shared/domain';
+import { EntityId, Course, Counted, SortOrder, PermissionService, Permission } from '@shared/domain';
```

Comment:
I think the PermissionService is the old way and now the AuthorizationService should be used.

Reply:
Done.